File: frentend/Trip_States_Management/packages/passenger/src/models/Passenger.ts
export type PassengerDataType = {
    Id:number;
     FirstName:string;
     LastName:string;
     MiddleName:string;
     LocalFirstName:string;
     LocalMiddleName:string;
     LocalLastName:string;
     BirthPlace:string;
     BirthCountry:string;
     BirthDay:Date;
     Nationality: string;
     Address1: string;
     Address2: string;
     ZipCode: string;
     City:string;
     CountryId:number;
     PhoneNumber1:number;
     PhoneNumber2:number;
  }

  class Passenger{
    public Id:number;
    public FirstName:string;
    LastName:string;
     MiddleName:string | undefined;
     LocalFirstName:string| undefined;
     LocalMiddleName:string| undefined;
     LocalLastName:string| undefined;
     BirthPlace:string
     BirthCountry:string;
     BirthDay:Date;
     Nationality: string;
     Address1: string;
     Address2: string;
     ZipCode: string;
     City:string;
     CountryId:number;
     PhoneNumber1:number| undefined;
     PhoneNumber2:number| undefined;
    
   
     public constructor(props: PassengerDataType){
         
        this.Id =  props.Id
        this.FirstName = props.FirstName
        this.LastName = props.LastName
        this.MiddleName = props.MiddleName 
        this.LocalFirstName=props.LocalFirstName;
        this.LocalMiddleName=props.LocalMiddleName;
        this.LocalLastName=props.LocalLastName;   
        this.BirthPlace = props.BirthPlace
        this.BirthCountry = props.BirthCountry
        this.BirthDay = props.BirthDay
        this.Nationality= props.Nationality
        this.Address1= props.Address1
        this.Address2= props.Address2
        this.ZipCode= props.ZipCode
        this.City= props.City
        this.CountryId= props.CountryId
        this.PhoneNumber1 = props.PhoneNumber1
        this.PhoneNumber2 = props.PhoneNumber2
    } 
}

export default Passenger;

File: backend/Apprilium.TripPlanner/Apprilium.TripPlanner.Application/Passenger/Model/Passenger.cs


using Apprilium.TripPlanner.Domain.Common;

namespace Apprilium.TripPlanner.Application.Model
{
    public class Passenger : PagingParameter
    {
        public int Id { get; set; }
        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string MiddleName { get; set; }

        public string LocalFirstName { get; set; }

        public string LocalMiddleName { get; set; }

        public string LocalLastName { get; set; }

        public DateTime BirthDay { get; set; }

        public string BirthPlace { get; set; }

        public string BirthCountry { get; set; }

        public string Nationality { get; set; }

        public string Address1 { get; set; }

        public string Address2 { get; set; }

        public string ZipCode { get; set; }

        public string City { get; set; }

        public int CountryId { get; set; }

        public int PhoneNumber1 { get; set; }

        public int PhoneNumber2 { get; set; }
    }
}
